mac/Focalboard/AppDelegate.swift

```diff
@@ -26,6 +26,13 @@ class AppDelegate: NSObject, NSApplicationDelegate {
 		NotificationCenter.default.post(name: AppDelegate.serverStartedNotification, object: nil)
 	}
 
+	func applicationShouldHandleReopen(_ sender: NSApplication, hasVisibleWindows flag: Bool) -> Bool {
+		if !flag {
+			openNewWindow(nil)
```

Comment:
If I understand [the docs](https://developer.apple.com/documentation/appkit/nsapplicationdelegate/1428638-applicationshouldhandlereopen?language=objc) correctly, it sounds like we should return false if we handle the open ourselves?

Reply:
Hm, I think you're right, yeah. I'm away for the weekend but I'll make the changes next week.

Reply:
Made the change.